prospector/disabled_tests/skip_test_prospector_database.py

```diff
@@ -66,26 +66,38 @@ def example_vulnerability_git_repo(example_vulnerability):
 
 @pytest.mark.database
 def test_timestamp_to_timestamp_interval(example_vulnerability):
-    assert database.timestamp_to_timestamp_interval(
-        example_vulnerability["nvd_published_timestamp"],
-        days_before=730,
-        days_after=100,
-    ) == ("1457823600", "1529532000")
-    assert database.timestamp_to_timestamp_interval(
-        str(example_vulnerability["nvd_published_timestamp"]),
-        days_before=730,
-        days_after=100,
-    ) == ("1457823600", "1529532000")
-    assert database.timestamp_to_timestamp_interval(
-        str(example_vulnerability["nvd_published_timestamp"]),
-        days_before="730",
-        days_after="100",
-    ) == ("1457823600", "1529532000")
-    assert database.timestamp_to_timestamp_interval(
-        str(example_vulnerability["nvd_published_timestamp"]),
-        days_before=0,
-        days_after=0,
-    ) == ("1520895600", "1520895600")
+    assert (
```

Comment:
not sure I see what has changed here; BTW, I would not change legacy_* files, they are there just for reference

Reply:
ah, ok, now I see: I guess this is black reformatting; that's ok